enterprise/api_client/xpert_ai.py

```diff
@@ -0,0 +1,30 @@
+"""
+Xpert client
+"""
+
+import json
+
+import requests
+
+from django.conf import settings
+
+
+def chat_completion(prompt, role):
+    """
+    Generate response using xpert api.
+    """
+    completion_endpoint = getattr(settings, 'CHAT_COMPLETION_API', None)
+    completion_endpoint_key = getattr(settings, 'CHAT_COMPLETION_API_KEY', None)
+
+    headers = {'Content-Type': 'application/json', 'x-api-key': completion_endpoint_key}
+
+    body = {'message_list': [{'role': role, 'content': prompt},]}
+
+    response = requests.post(
+        completion_endpoint,
+        headers=headers,
+        data=json.dumps(body),
+        timeout=(5, 20)  # connect and read timeout
```

Comment:
nit: we can put these timouts variables in settings file for better readabilioty and reuseability.